integration-tests/ci-visibility/unskippable-test/test-unskippable.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * @datadog {"unskippable": true}
```

Comment:
this is what users need to add to the suites that are not to be skipped. It's a JSON because it's more easily extendible (for example, for running multiple times to check flakiness)